### PR TITLE
remove out-of-date information from the site

### LINKED ISSUE #6: update to reflect the current state of Compliance Masonry
Seems to have fallen behind, unfortunately. We should also decide whether we want the Masonry and schema documentation to live [on the website](http://opencontrol.xyz/schema/) or whether we should just direct people to the respective repositories. The former is great for centralization/navigation purposes, whereas the latter is great for being easier to update alongside code changes.

/cc @jcscottiii

### PR DESCRIPTION
**Pull request by afeld**
Closes #6.

While there is likely information we will want to add/move to the site, this removes nearly everything on there that is out of date, which should help ease any confusion for people who stumble upon it. Also:
- Explain (briefly) how to generate the site locally
- Embed the video of @joshuamckenty's All Things Open keynote
- Fixed all the broken links I came across

/cc @brittag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,7 @@ Every deployment includes updates to continuous monitoring.
 > __Compliance as Code.__
 
 The full website is online at [http://opencontrol.xyz](http://opencontrol.xyz).
```
Comment (joshuamckenty) [inline README.md:24]: Maybe we should be promoting the http://open-control.org url instead - I've had push-back from some agencies on the .xyz TLD.

**Comment by joshuamckenty** [comment]
Merging as it's a huge step forward. Would consider another PR for changes to base URL.

**Comment by afeld** [comment]
How does this site get re-deployed? Not seeing the changes live yet.

**Comment by joshuamckenty** [comment]
I crashed the dragon concourse instances (which were running CD for the website) during an upgrade to an alpha version of concourse and BOSH. Still fixing.

**Pull request merged**